app/models/bet_recommendation.rb

```
@@ -0,0 +1,85 @@
+#  fk_rails_...  (game_id => games.id)
+#  fk_rails_...  (game_odd_id => game_odds.id)
+#  fk_rails_...  (prediction_id => predictions.id)
+#  fk_rails_...  (ratings_config_version_id => ratings_config_versions.id)
+#
+class BetRecommendation < ApplicationRecord
+  belongs_to :ratings_config_version
+  belongs_to :game
+  belongs_to :prediction
+  belongs_to :game_odd
+
+  validates :bet_type, uniqueness: { scope: %i[prediction_id game_odd_id] }
```

RuboCop (1.75.5)

[convention] Uniqueness validation should have a unique index on the database column.

(Rails/UniqueValidationWithoutIndex)